_drafts/wework-com-is-going-static.markdown

```diff
@@ -0,0 +1,113 @@
+
+![Performance graph of locations page](http://res.cloudinary.com/wework/image/upload/c_scale,f_auto,w_1000/v1448740493/engineering/locations-graph.png)
+
+So how are we serving up traffic to `wework.com/locations/` from the static host and other pages from our current non-static host? It was as simple has modiftying some DNS settings to route all traffic through Netlify, [creating a `_redirects` file](https://www.netlify.com/docs/redirects) at the root of our very basic static site, with some rules about which URL's to pass through and which ones to handle. Netlify's proxy and rewrite is actually very intelligent in how it handles requests. If it finds the file, folder or resource locally in your folder structure, it will serve that up before it tries to proxy it. On our site, we have a catch-all rule that looks like this:
+
+```coffeescript
+extensions: [
+  netlify
+    rewrites:
+      '/':  dubsUrl
+      '/*':  dubsUrl + '/:splat'
+```
+
+Which basically says, route all traffic through to `dubsUrl`, which is defined further up using `ENV` variables. We never have to touch this rule, and as soon as we added the `/locations/index.html` resource to our static site, Netlify was smart enough to serve that page vs proxying it through.
+
+
+The reality is that we are currently maintaining two versions of our site as we move things over, but this allows us to continue doing business as usual, make updates to existing pages and not have to stop our normal workflow as we migrate things over little by little. What's great about all of this is that we are sharing the same data across both sites.
+
+By doing this, we hope to see an improvement in direct and SEO traffic and hopefully a small uptick in our KPI numbers, as it has been proven that conversion numbers generally improve when your pages load faster and people are able to get to the information they are looking for quicker. This also allows us to expand globally without having to worry as much about traffic load and performance. One thing to keep in mind with all of this is that it is not necessarily *easier*, but it is a lot *simpler*. Also as a side effect of serving up static pages, your site becomes a lot more secure. There is a [great article on Smashing Magazine](http://www.smashingmagazine.com/2015/11/modern-static-website-generators-next-big-thing/) written by the co-founder of Netlify (Mathias Biilmann Christensen) about why static site generators are the next big thing. I highly recommend checking it out.
```

Comment:
**as it has been proven that conversion numbers generally improve when your pages load faster and people are able to get to the information they are looking for quicker.**

could reduce to: as it has been proven that conversion numbers generally improve when your pages load **more quickly**.